File: s08_Maps/MyApplication/app/src/main/java/com/omeryildizce/maps/roomdb/PlaceDatabase.java
package com.omeryildizce.maps.roomdb;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.omeryildizce.maps.model.Place;

@Database(entities = {Place.class}, version = 1)
public abstract class PlaceDatabase extends RoomDatabase {
    public abstract PlaceDao placeDao();

}

File: s21_Maps/KotlinMaps/app/src/main/java/com/omeryildizce/kotlinmaps/roomdb/PlaceDatabase.kt
package com.omeryildizce.kotlinmaps.roomdb

import androidx.room.Database
import androidx.room.RoomDatabase
import com.omeryildizce.kotlinmaps.model.Place

@Database(entities = arrayOf(Place::class), version = 1)
abstract class PlaceDatabase : RoomDatabase() {
    abstract fun placeDao() : PlaceDao
}